src/tests/debug-tests.c

```diff
@@ -48,10 +48,11 @@ START_TEST(test_debug_convert_old_level_old_format)
         SSSDBG_TRACE_FUNC,
         SSSDBG_TRACE_LIBS,
         SSSDBG_TRACE_INTERNAL,
-        SSSDBG_TRACE_ALL | SSSDBG_BE_FO
+        SSSDBG_TRACE_ALL | SSSDBG_BE_FO,
```

Comment:
If this is lowest possible debug level why it is not included in SSSDBG_TRACE_ALL?

Reply:
Not sure I understand the question.

This new debug level is "lower" than SSSDBG_TRACE_ALL.

I agree this sounds weird and I am open for suggestions.

Reply:
I think there is no easy way to solve this so lets stick with that.